dcicutils/env_utils.py

```diff
@@ -210,14 +210,10 @@ def is_hotseat_env(envname):
 
 
 def get_env_from_context(settings, allow_environ=ALLOW_ENVIRON_BY_DEFAULT):
```

Comment:
I would add a comment here for the next major version to remove this argument, as we don't want to make a potentially breaking change now but probably should in the future.